CMakeLists.txt

```diff
@@ -1,18 +1,26 @@
 # SPDX-License-Identifier: Apache-2.0
 
 # Match the minimum required version of LLVM and MLIR
-cmake_minimum_required(VERSION 3.20.0)
+cmake_minimum_required(VERSION 3.24.0)
```

Comment:
We use ubuntu:jammy for our dev docker image. Its cmake version is up to 3.22.1. So we cannot require more than that.

Reply:
> We use ubuntu:jammy for our dev docker image. Its cmake version is up to 3.22.1. So we cannot require more than that.

See, I leave this variation of proposal for a possible future betterment.

Thanks.

Reply:
> > We use ubuntu:jammy for our dev docker image. Its cmake version is up to 3.22.1. So we cannot require more than that.

I was mistaken , `FetchContent` [requires](https://cmake.org/cmake/help/latest/module/FetchContent.html) minimum `3.11` not `3.24` so it fits in the actual `3.22` requirement.
I unchanged  the version requirement from proposal with a new push.